docs/r2/tensorboard_scalars_and_keras_basic.ipynb

```diff
@@ -0,0 +1,452 @@
+        "\n",
+        "As you watch the training progress, note how both training and validation loss rapidly decrease and then remain stable. In fact, you could have stopped training after 25 epochs, because the training didn't improve much after that point.\n",
+        "\n",
+        "Hover over the graph to see specific data points. You can also try zooming in with your mouse, or selecting part of them to view more detail.\n",
+        "\n",
+        "Using the \"Runs\" selector on the left, you can select specific runs, or select only training or validation. As you continue to experiment and develop your model, you can compare runs by selecting only those you want to examine.\n"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "finK0GfYyefe",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "Ok, TensorBoard indicates that the metrics look good! Now see how the model actually behaves in real life. \n",
+        "\n",
+        "Given the input data (60, 25, 2), the line *y = 0.5x + 2* should yield (32, 14.5, 3). Does the model agree?"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "EuiLgxQstt32",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "print(model.predict([60, 25, 2]))"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "bom4MdeewRKS",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "Not bad!"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "vvwGmJK9XWmh",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "## Logging custom scalars\n",
+        "\n",
+        "What if you want to log custom values, such as a dynamic learning rate? To do that, you need to use the TensorFlow Summary API.\n",
+        "\n",
+        "Retrain the regression model and log a custom learning rate. Here's how:\n",
+        "\n",
+        "1.  Create a file writer, using ```tf.summary.create_file_writer()```.\n",
+        "2.  Define a custom learning rate function. This will be passed to the Keras [LearningRateScheduler](https://www.tensorflow.org/api_docs/python/tf/keras/callbacks/LearningRateScheduler) callback.\n",
+        "3.  Inside the learning rate function, use ```tf.summary.scalar()``` to log the custom learning rate.\n",
+        "4.  Pass the LearningRateScheduler callback to Model.fit().\n",
+        "\n",
+        "In general, to log a custom scalar, you need to use ```tf.summary.scalar()``` with a file writer. You can reuse the file writer for multiple scalars."
```

Comment:
The file writer corresponds to a specific run, right?

Reply:
It doesn't correspond to a run; it's a directory. Reworded.